src/main/kotlin/com/virtuslab/pulumikotlin/scripts/ComputeSchemaSubsetScript.kt

```diff
@@ -57,12 +60,24 @@ private const val HELP =
 
 class ComputeSchemaSubsetScript(outputStream: OutputStream = System.out) : CliktCommand(help = HELP) {
 
+    private val json = Json {
+        prettyPrint = true
+        ignoreUnknownKeys = true
+    }
     private val logger = KotlinLogging.logger {}
     private val printStream = PrintStream(outputStream)
 
     private val schemaPath: String by option().required()
-    private val context: Context by option().enum<ExposedContext>().convert { it.toContext() }.required()
-    private val name: String by option().required()
+    private val existingSchema: String? by option()
+    private val nameAndContexts: List<NameWithContext>
+        by option("--name-and-context")
+            .transformValues(2) { NameWithContext(it[0], ExposedContext.valueOf(it[1].capitalize()).toContext()) }
+            .multiple()
+            .help(
+                "Specify multiple names and their context (type / function). " +
+                    "For example --name-and-context VirtualMachine --name-and-context",
```

Comment:
Is this example correct? There's no context here and the second `--name-and-context` isn't followed by anything

Reply:
done, also updated other docs